lib/tanker/identity.rb

```diff
@@ -119,12 +125,14 @@ def self.get_public_identity(serialized_identity)
 
       if identity['target'] == 'email'
         identity['target'] = 'hashed_email'
-        public_identity['value'] = Base64.strict_encode64(Crypto.generichash(public_identity['value'], BLOCK_HASH_SIZE))
+        public_identity['value'] = Crypto.hashed_provisional_value(public_identity['value'])
+      elsif identity['target'] == 'phone_number'
+        public_identity['value'] = Crypto.hashed_provisional_value(public_identity['value'])
       end
```

Comment:
Do you want to be very explicit, or factoring would be possible?

```ruby
if identity['target'] == 'email'
  identity['target'] = 'hashed_email'
end

public_identity['value'] = Crypto.hashed_provisional_value(public_identity['value'])

Reply:
Done. Note that I have to wrap the second line in `if identity['target'] != 'user'`, because this can also be called on permanent identities

Reply:
Roger, copy that.